arm/Microsoft.DomainServices/AzureADDomainServices/deploy.bicep

```
@@ -4,62 +4,59 @@ param name string = domainName
 @sys.description('Required. The domain name specific to the Azure ADDS service.')
 param domainName string
 
-@description('Required: The name of the sku specific to Azure ADDS Services - Standard is the default')
+@description('Required. The name of the sku specific to Azure ADDS Services - Standard is the default')
 param sku string
 
-@description('Required: The subnet Id to deploy the Azure ADDS Services')
-param subnetId string
-
-@description('Required: The location to deploy the Azure ADDS Services')
+@description('Required. The location to deploy the Azure ADDS Services')
 param location string
 
-@description('Optional: Additional replica set for the managed domain')
+@description('Optional. Additional replica set for the managed domain')
 param replicaSets array = []
 
-@description('Required: The value is the base64encoded representation of the certificate pfx file')
+@description('Required. The value is the base64encoded representation of the certificate pfx file')
 param pfxCertificate string
 
-@description('Required: The value is to decrypt the provided Secure LDAP certificate pfx file')
+@description('Required. The value is to decrypt the provided Secure LDAP certificate pfx file')
 @secure()
 param pfxCertificatePassword string
 
-@description('Required: The email recipient value to receive alerts')
+@description('Optional. The email recipient value to receive alerts')
```

If there is no default value it makes the parameter mandatory.

```suggestion
@description('Required. The email recipient value to receive alerts')
```